.buildkite/premerge.definition.yaml

```diff
@@ -2,6 +2,7 @@ agent_queue_id: trigger-pipelines
 description: TestGyms premerge (calls GDK premerge)
 github:
   default_branch: master
+  pull_request_branch_filter_configuration: ["!docs*"]
```

Comment:
will have to `circle ci pipeline push` after this is merged. I agree it would be nice to have a checker/auto-pusher

Reply:
done